test/unit/cookie.test.js

```diff
@@ -287,19 +288,63 @@ describe('Cookie', function () {
     });
 
     describe('stringify', function () {
+        it('should return single Set-Cookie header string with key value', function () {
+            var cookie = new Cookie({
+                name: 'foo',
+                value: 'fooTest'
+            });
+
+            expect(Cookie.stringify(cookie))
+                .equals('foo=fooTest');
```

Comment:
Nitpick: `.to.equals` allows better readability. And we can keep it on the same line.

It's your choice to change it in this PR. But we can keep in mind for later PRs.

Reply:
Fixed.